structuredevent-service-legacy/src/main/java/com/sequenceiq/cloudbreak/structuredevent/service/telemetry/log/cluster/ClusterRequestedLogger.java

```diff
@@ -26,7 +32,10 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ClusterRequestedLogger.class);
 
-    private static final EnumSet<UsageProto.CDPClusterStatus.Value> TRIGGER_CASES = EnumSet.of(CREATE_STARTED, CREATE_FINISHED, CREATE_FAILED);
+    private static final EnumSet<UsageProto.CDPClusterStatus.Value> TRIGGER_CASES = EnumSet.of(CREATE_STARTED, CREATE_FINISHED, CREATE_FAILED,
```

Comment:
Are the RESUME/SUSPEND _STARTED/_FINISHED/_FAILED and the UPGRADE/UPSCALE/DOWNSCALE _STARTED  use cases intentionally excluded from the trigger cases?

Reply:
It is intentional. I am adding only those events that are mutating the DL/DH values.  The start/stop and upgrade started, scaling started events are changing only the state, therefore they are logged only to the state changed table.